src/plugins/data/public/plugin.ts

```diff
@@ -59,7 +59,9 @@ export class DataPublicPlugin
 
   constructor(initializerContext: PluginInitializerContext<ConfigSchema>) {
     this.searchService = new SearchService(initializerContext);
-    this.queryService = new QueryService();
+    this.queryService = new QueryService(
+      initializerContext.config.get().query.timefilter.minRefreshInterval
+    );
```

Comment:
Originally I did pass the `initializerContext` but since `QueryService` is also used independently in the `security_solution` plugin, I think passing the `minRefreshInterval` on it's own is best. Mainly because the config value is only in the  `initializerContext` of the `data` plugin and not `security_solution`.